lib/src/feedback_widget.dart

```diff
@@ -41,7 +43,7 @@ class FeedbackWidgetState extends State<FeedbackWidget>
   ScreenshotController screenshotController = ScreenshotController();
   TextEditingController textEditingController = TextEditingController();
 
-  bool isNavigatingActive = true;
+  bool isNavigatingActive = false;
```

Comment:
I would prefer to expose `ControleMode` https://github.com/ueman/feedback/blob/0758e6d06fe4da8a4156dc86503f431cf3154352/lib/src/controls_column.dart#L4 this instead. 

This is also a nice quality of life improvement. Thanks!

Reply:
This is how I implemented it beforehand, but it makes the usage really awkward. For this, a subclass needs to be imported and you need to look at the documentation to do this. With a boolean it is very easy to implement and follows the way it works internally. Once there are more modes, the internal way needs to be changed as well and I think then it would make sense to change this as well.